pkgs/by-name/go/go-mtree/package.nix

```
@@ -0,0 +1,35 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule rec {
+  pname = "gomtree";
+  version = "0.5.4";
+
+  src = fetchFromGitHub {
+    owner = "vbatts";
+    repo = "go-mtree";
+    rev = "v${version}";
+    hash = "sha256-MDX16z4H1fyuV5atEsZHReJyvC+MRdeA54DORCFtpqI=";
+  };
+
+  vendorHash = null;
+
+  # tests fail with nix due to ro file system
+  doCheck = false;
+
+  ldflags = [
+    "-s"
+    "-w"
+  ];
```

```suggestion
  ldflags = [
    "-s"
    "-X main.Version=${version}"
  ];
```

Maybe check if the other packages expect version injection as well.